packages/edn/edn.0.1.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "Andrew Rudenko <ceo@prepor.ru>"
+authors: "Andrew Rudenko <ceo@prepor.ru>"
+homepage: "http://github.com/prepor/ocaml-edn"
+bug-reports: "http://github.com/prepor/ocaml-edn/issues"
+license: "MIT"
+dev-repo: "http://github.com/prepor/ocaml-edn"
```

Comment:
Could you please change this to

`dev-repo: "http://github.com/prepor/ocaml-edn.git"`
?